README.md

```diff
@@ -1,4 +1,4 @@
-# MedSAM
+# MedSAM Breast Ultrasound Finetune
```

Comment:
Hi @FrexG ,

Thanks for your great pull. Could you please remove the change of the title?

Reply:
I have removed the title!